src/core/metrics/sources/nginx_access_log.go

```diff
@@ -471,4 +522,22 @@ func getDefaultCounters() map[string]float64 {
 		"v1_1":                    0,
 		"v2":                      0,
 	}
+
+	upstreamConnnectCounters := map[string]float64{
```

Comment:
Small spelling mistake 
```suggestion
	upstreamConnectCounters := map[string]float64{
```